src/routers/options.ts

```diff
@@ -3,10 +3,13 @@ import { default as express } from 'express';
 import type * as db from '../db';
 import { isLoggedIn } from '../middleware/isLoggedIn';
 import { getOption } from '../services/options';
+import { saveComment, getCommentsForOption } from '../services/comments';
 
 const router = express.Router();
 
 export function optionsRouter({ dbPool }: { dbPool: PostgresJsDatabase<typeof db> }) {
   router.get('/:optionId', isLoggedIn(), getOption(dbPool));
+  router.get('/:optionId/comment-data', isLoggedIn(), getCommentsForOption(dbPool));
```

Comment:
remove the `-data` from this endpoint

Reply:
changed the get and post url to `comments`